Rubberduck.Inspections/Abstract/InspectionBase.cs

```diff
@@ -166,5 +159,21 @@ public int CompareTo(object obj)
         {
             return CompareTo(obj as IInspection);
         }
+        protected abstract IEnumerable<IInspectionResult> DoGetInspectionResults();
```

Comment:
Don't do this if you don't have ReSharper (pretty sure Vanilla VS would make that a pain in the neck), but I'd have gone with `GetInspectionResultsInternal` here...

Reply:
Why can't he just rename it now?

Reply:
@Hosch250 because the rename needs to trickle down to *every single implementation* of the base class. Without R# I'm not sure vanilla-VS' rename refactoring gets it done. I'll merge now and PR the rename later.

Reply:
I'm pretty sure that VS vanilla gets this done. On that note: I tried to make GetInspectionResults `sealed` but that only errored, so I left it alone...